Makefile

```diff
@@ -19,3 +19,10 @@ package:
 .PHONY: clean
 clean:
 	rm -rf dist/ build/ kapitan.egg-info/
+
+.PHONY: pycodestyle
+pycodestyle:
```

Comment:
I would argue we should use flake8 instead.

Reply:
switched to flake8 :+1: